internal/services/devcenter/dev_center_resource.go

```diff
@@ -56,8 +57,12 @@ func (r DevCenterResource) Arguments() map[string]*pluginsdk.Schema {
 			Type:     pluginsdk.TypeString,
 		},
 		"resource_group_name": commonschema.ResourceGroupName(),
-		"identity":            commonschema.SystemAssignedUserAssignedIdentityOptional(),
-		"tags":                commonschema.Tags(),
+		"project_catalog_item_sync_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
```

Comment:
Since we're defaulting the value for this to `CatalogItemSyncEnableStatusDisabled`, could you add an explicit default of `false` to this property here and in the docs?

```suggestion
			Optional: true,
			Default: false,
```

Reply:
Updated